src/scripts/main.js

```diff
@@ -22,7 +22,19 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const key in data) {
+    const liElem = document.createElement('li');
+
+    liElem.textContent = key;
+    list.appendChild(liElem);
+
+    if (Object.keys(data[key]).length !== 0) {
```

Comment:
Replace with `Object.keys(data[key]).length`, it will be the same as `Object.keys(data[key]).length !== 0`